resources/lang/fr/messages.php

```diff
@@ -67,12 +70,12 @@
     'email_utility_description' => 'Vérifiez les paramètres de messagerie et envoyez des e-mails de test.',
     'entry_origin_instructions' => 'La nouvelle traduction héritera des valeurs qu’a l’entrée dans le site sélectionné.',
     'expect_root_instructions' => 'Considérer la première page de l’arborescence comme une page "racine" ou "d’accueil".',
-    'field_conditions_always_save_instructions' => 'Toujours enregistrer la valeur du champ, quelle que soit la façon dont les conditions du champ sont évaluées.',
     'field_conditions_instructions' => 'Quand afficher ou masquer ce champ.',
     'field_desynced_from_origin' => 'Désynchronisé de l’origine. Cliquez pour synchroniser et revenir à la valeur d’origine.',
     'field_synced_with_origin' => 'Synchronisé avec l’origine. Cliquez ou modifiez le champ pour désynchroniser.',
     'field_validation_advanced_instructions' => 'Ajouter des règles de validation plus avancées à ce champ.',
     'field_validation_required_instructions' => 'Permet de contrôler si ce champ est obligatoire ou non.',
+    'fields_always_save_instructions' => 'Toujours enregistrer la valeur du champ, quelle que soit la façon dont les conditions du champ sont évaluées.',
```

Comment:
Can you merge 3.4 back into this branch? It looks like an issue. The `fields_always_save_instructions` key was recently renamed to `field_conditions_always_save_instructions`. Here you are reverting that.

Reply:
I'll try to do that

Reply:
I see that maybe you're having trouble with Git here. Sorry I didn't mean to complicate it for you. I don't mind fixing it.

Reply:
yeah, I have never been very "git-savvy"...sorry about that. Is it OK now?